lib/active_model_serializers/adapter/json_api/relationship.rb

```diff
@@ -34,6 +34,11 @@ def as_json
 
         private
 
+        def ensure_valid!(json)
```

Comment:
@remear it's the difference between "we're not saying anything about this relationship one way or the other" versus "this relationship is blank".

For instance, let's say you use ember-data and `post` belongs to `author`, both objects already loaded into memory. A subsequent GET to the API that returns `author: { data: null }` will blow away `post.author`. Omitting the data key does not.